From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:24 +0800
Subject: en_IE-am_pm

# Description: Add am_pm for en_IE.
# Ubuntu: https://launchpad.net/bugs/199899
# Upstream: http://sourceware.org/bugzilla/show_bug.cgi?id=3236

# Description: Add am_pm for en_IE.
# Ubuntu: https://launchpad.net/bugs/199899
# Upstream: http://sourceware.org/bugzilla/show_bug.cgi?id=3236
---
 localedata/locales/en_IE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localedata/locales/en_IE b/localedata/locales/en_IE
index d876e10..fe46ba5 100644
--- a/localedata/locales/en_IE
+++ b/localedata/locales/en_IE
@@ -111,7 +111,7 @@ d_t_fmt     "%a %d %b %Y %T"
 date_fmt    "%a %d %b %Y %T %Z"
 d_fmt       "%d//%m//%y"
 t_fmt       "%T"
-am_pm       "";""
+am_pm       "AM";"PM"
 t_fmt_ampm  ""
 first_weekday 2
 week 7;19971130;4
